draft-ietf-quic-multipath.md

```diff
@@ -307,6 +307,24 @@ to migrate to that path.  Instead, servers SHOULD consider new paths
 over which non-probing packets have been received as available
 for transmission.
 
+
+## Path State Management
+
+An endpoint uses PATH_STATUS frames to inform that the peer should
+send packets in the preference expressed by these frames.
```

Comment:
I think I would be useful to explicitly state that these are indications, i.e., the peer might not follow the endhost advertisements.

Reply:
Added note for that.